core/admin/mailu/internal/views/autoconfig.py

```diff
@@ -0,0 +1,184 @@
+from mailu.internal import internal
+
+from flask import current_app as app
+import flask
+import xmltodict
+
+@internal.route("/autoconfig/mozilla")
+def autoconfig_mozilla():
+    # https://wiki.mozilla.org/Thunderbird:Autoconfiguration:ConfigFileFormat
+    hostname = app.config['HOSTNAME']
+    xml = f'''<?xml version=\"1.0\"?>
```

Comment:
Escaping the double quotes is not necessary in this and the following f-strings.

Reply:
That's ce9dc3a3